apps/src/templates/ContainedLevel.jsx

```diff
@@ -31,7 +33,37 @@ const ContainedLevel = React.createClass({
       codeStudioLevels.lockContainedLevelAnswers();
     } else {
       // No answers yet, disable Run button until there is an answer
-      $('#runButton').prop('disabled', true);
+      let runButton = $('#runButton');
+      runButton.prop('disabled', true);
+      // Disabled buttons don't trigger mouse events, add a dummy element to
+      // receive the click and trigger the display of the callout.
+      let clickReceiver = $('<div id="clickReceiver"/>');
+      let boundingClientRect = runButton.get(0).getBoundingClientRect();
+      clickReceiver.css({
+        height: boundingClientRect.height + 'px',
+        width: boundingClientRect.width + 'px',
+        position: 'absolute',
+        top: 0,
+      });
+      $('#gameButtons').append(clickReceiver);
+      clickReceiver.bind('click',
+        () => $(window).trigger('attemptedRunButtonClick'));
```

Comment:
Ah never mind, I see that the Run Button is disabled.  I wonder if [capturing events on the parent element](https://stackoverflow.com/a/32059136) would be cleaner?  Or not disabling the button, but bailing out early in the click handler if there's no answer selected.